util/wren.mk

```diff
@@ -103,9 +103,13 @@ else
 	SHARED_LIB_FLAGS := -Wl,-soname,$@.so
 	SHARED_EXT := so
 
+ifeq (,$(findstring Windows,$(OS)))
```

Comment:
>  I don't know the toolchain or make that set %OS% as mingw32.

`OS` isn't coming from the environment. It's defined by the makefile on line 81. :)